docs/faq.rst

```diff
@@ -987,6 +987,12 @@ stable. If the directory where you mount a filesystem is different every time,
 Borg assumes they are different files. This is true even if you backup these
 files with relative pathnames - borg uses full pathnames in files cache regardless.
 
+It is possible for some filesystems, such as ``mergerfs`` or network-mounted
+filesystems, to return inconsistent inode numbers across runs, causing borg to
+consider them changed. A workaround is to set the option ``--files-cache=ctime,size``
+to exclude the inode from the default cache calculation so that files with different
```

Comment:
... to exclude the ``inode`` number comparison from the files cache check so that files with different inode numbers ...